Packs/RecordedFuture/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "Recorded Future Intelligence",
     "description": "Recorded Future App, this pack is previously known as 'RecordedFuture v2'",
     "support": "partner",
-    "currentVersion": "1.7.13",
+    "currentVersion": "1.7.14",
```

```suggestion
    "currentVersion": "1.8.0",
```
Due to the extensive changes here, I think the version should be locked to 1.8.0 (not required, at your discretion, and the RN file name too).

---

Will do